spec/features/admin/admin_deletes_user_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+feature 'admin deletes user', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.